src/main/java/ooo/oshi/util/ParseUtil.java

```diff
@@ -1209,6 +1209,16 @@ public static List<String> parseByteArrayToStrings(byte[] bytes) {
         } while (end++ < bytes.length);
         return strList;
     }
+    
+    /**
+     * Parse a null-delimited byte array to a UTF-16 string.
+     *
+     * @param bytes A byte array containing a UTF-16 string 
+     * @return the UTF-16 string
+     */
+    public static String parseByteArrayToUtf16(byte[] bytes) {
+        return new String(bytes, StandardCharsets.UTF_16LE).trim();
```

Comment:
This probably works if the memory is zeroed before being populated.  I'm curious if it works for a null-terminated String with characters after the null.   OK with this for now, should consider whether to map WinAPI `WideCharToMultiByte` or convert this String to `char[]` and iterate looking for `\0` ... ideally Java would just create a function for us that dealt with this natively!